Base/usr/share/man/man1/xargs.md

```diff
@@ -0,0 +1,34 @@
+## Name
+
+xargs - build and execute commandlines from input
+
+## Synopsis
+
+```**sh
+$ xargs [options...] command [initial-arguments...]
+```
+
+## Description
+
+`xargs` reads items from a stream, delimited by some blank character (`delimiter`), and executes the `command` as many times as there are items, with any processed `initial-arguments`, possibly followed by an item read from the input.
+Each argument in `initial-arguments` is processed by replacing any occurence of the `placeholder` with the input item, and treating the entire resulting value as _one_ argument.
+If no argument in `initial-arguments` contains the `placeholder`, an argument is added at the end of the list containing only the `placeholder`.
+
+`xargs` will read the items from standard input by default, and when data is read from standard input, the standard input of `command` is redirected to `/dev/null`.
```

Comment:
`#include <english-is-not-my-first-language.h>`

stdin cannot be redirected _to_, only _from_

I think it sounds best to say "the standard input is redirected _to read_ from `/dev/null`